app/views/collection_drafts/forms/_acquisition_information.html.erb

```diff
@@ -1,7 +1,7 @@
 <!-- Platforms [r] -->
 <fieldset class="eui-accordion is-closed" id="platforms">
   <div class="eui-accordion__header">
-    <h3 class="eui-accordion__title eui-required-o">Platforms</h3>
+    <h3 class="eui-accordion__title eui-required-o always-required">Platforms</h3>
```

Comment:
So, I did not check and did not realize that collections did it this way. I may not have insisted on adding `always-required` in that case, but I still think this makes it more clear.